coremltools/converters/mil/mil/passes/tests/test_passes.py

```diff
@@ -7794,7 +7797,8 @@ def prog(x):
             y = x8
 
             if with_affine:
-                y = mb.add(x=y, y=np.random.rand(1,shape[1],1,1))
+                x9 = mb.add(x=y, y=np.random.rand(1,shape[1],1,1))
```

Comment:
nit: (1, shape[1], 1, 1)?

Reply:
I added spaces for all of these `rand(...)` calls in the new tests.